api/services/namespace.go

```diff
@@ -64,12 +64,19 @@ func (s *service) CreateNamespace(ctx context.Context, req *requests.NamespaceCr
 			ConnectionAnnouncement: "",
 		},
 		TenantID: req.TenantID,
+		Type:     models.NewDefaultType(),
 	}
 
 	if envs.IsCommunity() {
 		ns.Settings.ConnectionAnnouncement = models.DefaultAnnouncementMessage
 	}
 
+	if models.IsTypeTeam(models.Type(req.Type)) {
```

Comment:
I think that this helper functions could receive `req.Type` directly without the convention.